agot-bg-website/agotboardgame_main/views.py

```diff
@@ -223,6 +223,7 @@ def games(request):
         game.owner = request.user
         game.save()
 
+        # Why this length check? I always wondered why I just sometimes were redirected to the game automatically...
```

Comment:
Can you help me here. I'd like to change it to `if len(name) < 4 or 200 < len(name)` (the real max_length value)...